cwlgen/requirements.py

```diff
@@ -210,15 +210,16 @@ def parse_dict(cls, d):
 # }
 
 
-class SoftwareRequirement(Requirement):
+class 
+ment(Requirement):
```

Comment:
Hi @illusional 

I suppose this is a typo?

Cheers
M

Reply:
Yes :(